src/librustc_codegen_llvm/back/lto.rs

```diff
@@ -525,10 +566,34 @@ fn thin_lto(cgcx: &CodegenContext<LlvmCodegenBackend>,
             }));
         }
 
+        // Save the accumulated ThinLTO import information
```

Comment:
`// Save the current ThinLTO import information for the next compilation session`

Reply:
will do